projects/packages/publicize/webpack.config.js

```diff
@@ -1,11 +1,28 @@
 const path = require( 'path' );
 const jetpackWebpackConfig = require( '@automattic/jetpack-webpack-config/webpack' );
 
+/**
+ * @type {import('webpack').Configuration[]} Webpack configuration.
+ */
 module.exports = [
 	{
 		entry: {
 			[ 'classic-editor-share-limits' ]: './src/js/classic-editor-share-limits.js',
 			[ 'classic-editor-connections' ]: './src/js/classic-editor-connections.js',
+			social: {
```

Comment:
If you do like this, you shouldn't need the `filename:` below.
```suggestion
			[ 'jetpack-publicize' ]: {
```
That'll also make the CSS filenames match the JS filenames. As it is I think they won't get registered by `Assets::register_script()` due to the name mismatch.

Reply:
Ah Thanks